src/MockBot.Api/Models/Chat.cs

```diff
@@ -12,6 +12,8 @@ public class Chat
 
         [Required]
         public Collection<Message> Messages { get; } = new Collection<Message>();
+
+        public DateTime CreatedAt { get; } = DateTime.Now;
```

Comment:
This is a boring comment - but DateTime.UtcNow, just to keep telemetry in the UTC timezone?

Reply:
Small but important, thanks!